src/harness/externalCompileRunner.ts

```diff
@@ -27,9 +28,32 @@ abstract class ExternalCompileRunnerBase extends RunnerBase {
         const testList = this.tests && this.tests.length ? this.tests : this.enumerateTestFiles();
 
         describe(`${this.kind()} code samples`, () => {
+            const cwd = path.join(Harness.IO.getWorkspaceRoot(), this.testDir);
+            const placeholderName = ".node_modules";

```

Comment:
why not just add a `types: [..]` list?

Reply:
They have one already. That only says which things get auto-included without even doing an `import`, but the issue here is when i say `const x = require("chalk")` and we walk up the tree and find ours when we can't find it locally. 🌞